### PR TITLE
Change review assignment action (no automated follow-up review request)

### LINKED ISSUE #148: Reviewers Being Reassigned after force push
## The problem

@kaitj, I don't think this is a huge issue, really just a quality-of-life thing. Looks like right now when I make a force push on to an existing PR (especially after rebasing the branch onto new changes in `main`), GH refreshes all review requests (see the history of #138 for example). Is it possible and easy to set it so that only the initial pull request triggers the review request?

### PR DESCRIPTION
**Pull request by kaitj**
## Proposed changes

Resolves #148. Changes such that the reviewer assignment will trigger only if there was no assignee to the pull request. If a PR needs to be reviewed again (due to changes), users will have to manually request.

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you are unsure about any of the choices, don't hesitate to ask!

- [x] Changes have been tested to ensure that fix is effective or that a feature works.
- [x] Changes pass the unit tests
- [ ] I have included necessary documentation or comments (as necessary)
- [x] Any dependent changes have been merged and published

## Notes
All PRs will undergo the unit testing before being reviewed. You may be requested to explain or make additional changes before the PR is accepted.

### CODE REVIEW

**Review by pvandyken — APPROVED** [review]
Thanks, I personally prefer it this way

**Comment by kaitj** [comment]
It's a good call - must have missed the conditional when I moved it to it's own .yml file

File: .github/workflows/assign.yml [hunk 1]
```diff
@@ -6,9 +6,10 @@ on:
 jobs: 
   assign:
     runs-on: ubuntu-latest
+    if: github.event.pull_request.assignee == null
```
Comment (tkkuehn) [inline .github/workflows/assign.yml:9]: Do we want `github.event.pull_request.requested_reviewers` here? Assignee refers to the person who's working on the PR, I think.

Comment (tkkuehn) [reply .github/workflows/assign.yml:9]: Ah okay I forgot that this action also sets the assignee. No need to change it then!

**Pull request merged**